packages/vector_graphics/test/vector_graphics_test.dart

```diff
@@ -168,6 +169,44 @@ void main() {
 
     expect(testBundle.loadKeys, <String>['foo.svg', 'bar.svg']);
   });
+
+  testWidgets('Can update SVG picture', (WidgetTester tester) async {
+    final TestAssetBundle testBundle = TestAssetBundle();
+
+    await tester.pumpWidget(
+      DefaultAssetBundle(
+        bundle: testBundle,
+        child: const VectorGraphic(
+          loader: AssetBytesLoader('foo.svg'),
+        ),
+      ),
+    );
+    await tester.pumpAndSettle();
+
+    expect(tester.layers, contains(isA<PictureLayer>()));
+
+    await tester.pumpWidget(
+      DefaultAssetBundle(
+        bundle: testBundle,
+        child: const VectorGraphic(
+          loader: AssetBytesLoader('bar.svg'),
+        ),
+      ),
+    );
+    await tester.pumpAndSettle();
+
+    expect(tester.layers, contains(isA<PictureLayer>()));
+  });
+
+  testWidgets('PictureInfo.dispose is safe to call multiple times',
```

Comment:
Isn't this usually an error though? Normally the pattern should be `foo?.dispose(); foo = null;`

Reply:
I think its better for these APIs to be idempotent if they can be, but I don't have a strong opinion. its more work to make it an error though

Reply:
Yeah we don't have to make it error